airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/manifest.yaml

```diff
@@ -95,20 +105,23 @@ definitions:
         type: RequestOption
         field_name: start_date
         inject_into: request_parameter
+      end_datetime:
+        type: MinMaxDatetime
+        datetime: >-
+          {{ format_datetime(config['end_date'] if config['end_date'] else now_utc(), '%Y-%m-%dT%H:%M:%SZ') }}
```

Comment:
I can not see any end_date in `spec`.
If this field is used in our test config files only, you better use safe `get` method

Reply:
I don't know what happened. I'll add it back as it was there before.

Reply:
Please use config.get('end_date') instead